locales/zh_CN/LC_MESSAGES/duckduckgo.po

```diff
@@ -1909,8 +1909,8 @@ msgctxt "showcase_donate"
 msgid "Join our $500,000 Privacy Challenge"
 msgstr "加入我们的五十万美金募捐行动"
 
-msgid "Just a few clicks to install, and you're ready to go. It's that easy."
-msgstr "几次点击即可完成安装，就这么简单。"
+msgid "Just a few clicks to install,%sand you're ready to go.%sIt's that easy."
+msgstr "几次点击即可完成安装，％s就可以开始使用。％s就这么简单。"
```

Comment:
@tagawa would you be able to verify this is correct?

I used Google translate to compare the before/after and they seem to be correct. Now that we have placeholders (`%s`) in the string, we need to have the same number of placeholders in the translation, so I've extended the translation to allow for this.

Reply:
/cc @b9a1 -- if you could signoff on this or suggest a better translation, that would be great!

Reply:
Yes, position of the placeholders and translation looks good to me.

Reply:
```suggestion
msgstr "几次点击完成安装，%s即可开始使用。%s就这么简单。"
```
I would propose this as it’s more concise. Also note that the percent signs were ‘translated’ to fullwidth ones (U+FF05) when they shouldn’t.

Reply:
Oh, well spotted @b9a1 - thank you.

Reply:
@b9a1 thanks a lot!